microsoft-edge/webview2/webview2-api-reference.md

```diff
@@ -22,19 +22,20 @@ For more information, see [Overview of Microsoft Edge WebView2](index.md) and [G
 
 WebView2 is available for the following languages and frameworks:
 
-* [Win32 C++](/microsoft-edge/webview2/reference/win32/index)
-
 * .NET
```

Comment:
@david-risney @jasonstephen15 - Please check the platform naming, link text, I'd really like to specify C# or C++ on each link.  

This key section will coordinate/propagate to each article, eg an h2 API Reference or API Reference overview section at bottom of many articles - any wv2 articles that mention specific API types or members.  Need to normalize & propagate such links & link text & plat/lang labels.

Rendered: https://review.docs.microsoft.com/en-us/microsoft-edge/webview2/webview2-api-reference?branch=pr-en-us-2043#languages-and-frameworks